migrator/main.go

```diff
@@ -56,26 +57,32 @@ func run() error {
 		return nil
 	}
 
-	// TODO: ROX-9884, ROX-10700 -- turn off replicas and migrations until Postgres updates complete.
-	if !features.PostgresDatastore.Enabled() {
-		dbm, err := replica.Scan(migrations.DBMountPath(), conf.Maintenance.ForceRollbackVersion)
-		if err != nil {
-			return errors.Wrap(err, "fail to scan replicas")
-		}
+	// TODO: ROX-9884, ROX-10700 -- need to work on replicas and migrations
+	dbm, err := replica.Scan(migrations.DBMountPath(), conf.Maintenance.ForceRollbackVersion)
+	if err != nil {
+		return errors.Wrap(err, "fail to scan replicas")
+	}
 
-		replicaName, replicaPath, err := dbm.GetReplicaToMigrate()
-		if err != nil {
-			return err
-		}
-		option.MigratorOptions.DBPathBase = replicaPath
-		if err = upgrade(conf); err != nil {
-			return err
-		}
-		if err = dbm.Persist(replicaName); err != nil {
-			return err
-		}
-	} else {
-		gormDB, err := postgreshelper.Load(conf)
+	replicaName, replicaPath, err := dbm.GetReplicaToMigrate()
+	if err != nil {
+		return err
+	}
+
+	option.MigratorOptions.DBPathBase = replicaPath
+
+	// Close when needed
+	defer postgreshelper.Close()
```

Comment:
hmm why does this need to be here if postgres is disabled?

Reply:
Added condition of Postgres flag. Thanks!